### PR TITLE
Local environment

### LINKED ISSUE #116: Setup local test environment
Maybe, we can setup local test environment with ganache or something.

### PR DESCRIPTION
**Pull request by moreal**
This pull request tries to resolve #116.

- Run `docker-compose up -d`
- Run `./prepare-development.sh` and pastes the output into your `.env` (copied from `.env.example`
- Get and fill `SLACK_WEB_TOKEN`
- [ ] Transfer WNCG contract ownership to KMS address.
- Let's develop the bridge application.

### CODE REVIEW

**Comment by moreal** [comment]
I tried to use ganache but it throws an error with a message like `invalid signature v`. I gave up using it and deployed a contract at Ropsten network.

The last stack:
- Ethereum network: Ropsten network
- 9c network: local 9c-headless
- KMS: development KMS.

File: contracts/truffle-config.js [hunk 1]
```diff
@@ -5,9 +5,12 @@ const HDWalletProvider = require("@truffle/hdwallet-provider");
 module.exports = {
   networks: {
     development: {
-      host: "127.0.0.1",
-      port: 7545,
-      network_id: "*"
+      provider: () => new HDWalletProvider({
+        mnemonic: "two artist ribbon scene clown bachelor rail ivory grant budget clutch wrong",
```
Comment (longfin) [inline contracts/truffle-config.js:9]: Is this mnemonic usable? (or, is it okay on reveal here?)

File: prepare-development.sh [hunk 2]
```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Deploy WNCG contract.
+pushd contracts > /dev/null
+    rm -rf build
+    yarn
+    yarn truffle migrate --network development
+    contract_address=`jq '.networks."7777".address' ./build/contracts/WrappedNCG.json | tr -d '"'`
```
Comment (longfin) [inline prepare-development.sh:10]:
```suggestion
    contract_address=$(jq '.networks."7777".address' ./build/contracts/WrappedNCG.json | tr -d '"')
```

File: contracts/truffle-config.js (hunk 1 above)
Comment (moreal) [reply contracts/truffle-config.js:9]: Yes, I generated it for the test. I just hope others do not use the mnemonic code as his main wallet 😅

**Pull request merged**